docs/nutanix-prerequisites.md

```diff
@@ -0,0 +1,30 @@
+# Nutanix Platform Pre-Requisites
+
+In order to successfully use the Windows Machine Config Operator (WMCO) on a Nutanix Platform, 
+the following pre-requisites are required:
+
+* Upload a [VM image with a compatible Windows Server version](wmco-prerequisites.md#supported-windows-server-versions) to the Prism-Central/Prism-Element where the Machine VMs will be created with.
+* Add a [DNS entry](#adding-a-dns-entry-in-nutanix-environment) for the internal API endpoint in the Nutanix environment
+
+## Adding a DNS entry in Nutanix environment
+
+WMCO downloads the ignition files from the internal API server endpoint. To enable this, add a new DNS entry for
```

Comment:
Add a noun after "this" so that the reader and any translators will know what "this" refers to. For example:

"To enable this download" or "To enable these files" or "To enable this feature"